CONTRIBUTING.md

```diff
@@ -1,41 +1,43 @@
 # How To Contribute
-Contributing to Gaia Dimension has various ways to contribute to the development. From a standpoint, contributions can include Coding, Assets and Testing. Please note that any PRs made can and will be rejected if they do not comply with the following guidelines. PRs will be closed if:
+There are various ways to contribute to the development of the Gaia Dimension mod. Some common types of contributions can include donating code, making textures, translating, and testing. Please note that any Pull Requests (PRs) made can and will be rejected if they do not comply with the following guidelines.
 
-* Does not comply with the listed guielines
-* No action is done to resolve conflicts or build failures within 2 weeks
-
-## Coding
-Though it is not easy from a surface level, it's worth more to contribute via coding than any other place. However, there are a few things to note when submitting a PR for coding.
+## Code
+Though it is not easy, donating code is the best way to contribute. However, there are a few things to note when submitting a PR with code.
 
 ### Make sure braces are on *the same line*
 As much as some people are inclined to, and for consistency's sake, make sure your code is formatted to use same-line braces. This is to reduce on file size and better vertical readability.
 
 Example:
-```
+```java
 public void() {
 }
 ```
 ### It needs to work
 You can submit a PR that's got a solution to a problem or a new feature, but it needs to build. This can be checked by submitting the PR's and looking at the mergability. This repo uses Travis-CI, so builds are often checked.
 That being said, it also needs to be functional in-game. You could submit a PR based on the grounds that it works, but thorough testing is required to make sure that it doesn't break any other feature.
 
-## Assets
-Though they are fundamental, contributing to these isn't as desired due to how easy it is to contribute this way. Just like Coding, there are a few guidelines that need to be followed.
+## Textures
```

Comment:
I would keep this to "assets" as it also covers sound effects. That being said, I will need to draft up a paragraph for contributing sounds, as I've not made an attempt on this.

Reply:
While "assets" is the technical term, people who do not have a clue about Java will not know what it means. (Even though they should...)

Reply:
An update of what we should do about this?